key/aziot-key-test-common/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "aziot-key-test-common"
```

Comment:
It isn't really aziot-key-specific. You could just call it test-common and put it in the root of the repo.

Reply:
Fixed